students/12th/sunghomoon/project_westagram/user/views.py

```diff
@@ -0,0 +1,61 @@
+import json
+from django.views import View
+from django.http  import JsonResponse
+from .models      import User
+
+class SignUp(View):
+  def post(self, request):
+    data = json.loads(request.body)
+
+    UserData = User(
```

Comment:
클래스의 인스턴스라 할 지라도 변수이기 때문에 클래스 명명법을 사용하시면 헷갈릴 수 있습니다.
```suggestion
    user_data = User(
```